docs/release_notes/4.8.0.rst

```diff
@@ -0,0 +1,5 @@
+Features
```

Comment:
Instead of creating a release note you should add a news fragment like this https://github.com/VUnit/vunit/blob/master/docs/news.d/776.feature.rst

The number is the number of this PR and I would call this a feature, i.e. `1025.feature.rst`. The release notes will be automatically generated from the news fragments when we build the next release.